### ISSUE TITLE
Sentry could not resolve the “abort” exception,I have uploaded the Debug Symbols Successful (flutter packages pub run sentry_dart_plugin)

### ISSUE THREAD

**Issue by WangQiangQuan** [getsentry/sentry-dart#1539]
### Platform

Flutter Mobile

### Obfuscation

Enabled

### Debug Info

Enabled

### Doctor

Flutter (Channel stable, 3.3.1, on macOS 12.6 21G115 darwin-arm (Rosetta),

### Version

7.7.0

### Steps to Reproduce

Run flutter build apk before executing the sentry_dart_plugin plugin.

refer to: https://docs.sentry.io/platforms/flutter/upload-debug/

### Expected Result

flutter Correct stack

### Actual Result

libc++
+0x04a11c
<unknown>
libc++
+0x04a25c
<unknown>
libc++
+0x04afa0
<unknown>
libc++
+0x04af24
std::terminate
libc++
+0x0a4770
std::__1::thread::~thread
libc
+0x0f9124
__cxa_finalize
libc
+0x0eb4b8
exit
libflutter
+0x63174c
<unknown>
libflutter
+0x633348
<unknown>
libflutter
+0x6eff00
<unknown>
libflutter
+0x6efc94
<unknown>
libapp
+0x317084
_kDartVmSnapshotInstructions
libapp
+0x5a7860
_kDartIsolateSnapshotInstructions
libapp
+0x5a780c
_kDartIsolateSnapshotInstructions
libapp
+0x5a75e4
_kDartIsolateSnapshotInstructions
libapp
+0x868da8
_kDartIsolateSnapshotInstructions
libapp
+0x6b1ebc
_kDartIsolateSnapshotInstructions
libapp
+0x63cf30
_kDartIsolateSnapshotInstructions
libapp
+0x7e53f4
_kDartIsolateSnapshotInstructions
libapp
+0x749254
_kDartIsolateSnapshotInstructions
libapp
+0x7a174c
_kDartIsolateSnapshotInstructions
libapp
+0x447398
_kDartIsolateSnapshotInstructions
libapp
+0x447124
_kDartIsolateSnapshotInstructions
libapp
+0x360bf0
_kDartIsolateSnapshotInstructions
libapp
+0x3607cc
_kDartIsolateSnapshotInstructions
libapp
+0x3606c4
_kDartIsolateSnapshotInstructions
libapp
+0x360208
_kDartIsolateSnapshotInstructions
libapp
+0x4ab680
_kDartIsolateSnapshotInstructions
libapp
+0x4ab56c
_kDartIsolateSnapshotInstructions
libapp
+0x32ffc4
_kDartIsolateSnapshotInstructions
libapp
+0x32fef8
_kDartIsolateSnapshotInstructions
libapp
+0x32fe70
_kDartIsolateSnapshotInstructions
libapp
+0x316d7c
_kDartVmSnapshotInstructions
libflutter
+0x6cbd14
<unknown>
libflutter
+0x804908
<unknown>
libflutter
+0x5a0000
<unknown>
libflutter
+0x5a22ec
<unknown>
libflutter
+0x5ae734
<unknown>
libflutter
+0x3460cc
<unknown>
libflutter
+0x349658
<unknown>
libutils
+0x0181a8
android::Looper::pollInner
libutils
+0x017d1c
android::Looper::pollOnce
libandroid
+0x018598
ALooper_pollOnce
libflutter
+0x349764
<unknown>
libflutter
+0x347d8c
<unknown>
libc
+0x0f55c8
<unknown>
libc
+0x08efbc
<unknown>

### Are you willing to submit a PR?

None

**Comment by krystofwoldrich** [comment]
Hi,
thank you for the message, we will take a look into it.

**Comment by WangQiangQuan** [comment]
okey  krystofwoldrich
If there is any progress, please feel free to contact me.
thank you!

**Comment by krystofwoldrich** [comment]
@WangQiangQuan Could you share a code snippet on how to reproduce the error? That would help us with debugging.

**Comment by WangQiangQuan** [comment]
sorry,
I don't know how to reproduce this problem. The purpose of using sentry is to find out what the problem is and solve it

**Comment by marandaneto** [comment]
@WangQiangQuan can you share your SDK init code snippet and the command you use to build your app and run the dart plugin?
Also, can you share the `sentry` config in your `pubspec` file? remove the tokens etc of course :)
Also, is this Android your iOS? look at this as well https://docs.sentry.io/platforms/flutter/troubleshooting/#native-symbolication-on-iosmacos

**Comment by WangQiangQuan** [comment]
@marandaneto 
I share a list of your questions below：

1: My SDK init code is like this
void main() {
const String flutterDSN = “my DSN”;
String env = "Dev";

await SentryFlutter.init(
  (options) {
    options.dsn = flutterDSN;
    options.environment = env;
    options.sampleRate = 1.0;
    options.attachScreenshot = true;
    options.screenshotQuality = SentryScreenshotQuality.medium;
    options.attachViewHierarchy = true;
  },
  appRunner: () {
    ...
  },
);

}

2 : the command I use to build My app and run the dart plugin is

flutter build apk 
And
flutter packages pub run sentry_dart_plugin

3: pubspec file is
sentry:
  upload_debug_symbols: true
  upload_source_maps: false
  upload_sources: false
  project: home-edu
  org: sentry
  auth_token:  (my token)
  url: http://sentry.staff.xdf.cn
  wait_for_processing: false
  log_level: error
  commits: auto
  ignore_missing: true

4: this problem is found on android device

**Comment by marandaneto** [comment]
@WangQiangQuan https://docs.sentry.io/platforms/flutter/upload-debug/#when-to-upload
You don't even need to upload debug symbols if you are not using `split-debug-info`.
Not sure whats your problem is about, can you share a link to an issue on sentry.io so I can take a look?

**Comment by WangQiangQuan** [comment]
@marandaneto  
https://sentry.staff.xdf.cn/organizations/sentry/issues/81000/?project=51&query=is%3Aunresolved&statsPeriod=14d

**Comment by marandaneto** [comment]
> sentry.staff.xdf.cn/organizations/sentry/issues/81000/?project=51&query=is%3Aunresolved&statsPeriod=14d

I don't have access to your self-hosted version.
Can you try to do the very same thing on sentry.io (you can create an organization test).

**Comment by WangQiangQuan** [comment]
i will try later


Can you spot the problem from this screenshot









Magic功名
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "getsentry/sentry-dart"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年7月7日(星期五) 下午5:21
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [getsentry/sentry-dart] Sentry could not resolve the “abort” exception,I have uploaded the Debug Symbols Successful (flutter packages pub run sentry_dart_plugin) (Issue #1539)





  
sentry.staff.xdf.cn/organizations/sentry/issues/81000/?project=51&amp;query=is%3Aunresolved&amp;statsPeriod=14d
  
I don't have access to your self-hosted version.
 Can you try to do the very same thing on sentry.io (you can create an organization test).

**Comment by krystofwoldrich** [comment]
@WangQiangQuan The screenshot didn't upload via the email response.

**Comment by WangQiangQuan** [comment]
@krystofwoldrich Please give me an email address and I'll send the screenshot

**Comment by krystofwoldrich** [comment]
@WangQiangQuan Can you upload it here? That would be better to keep all the context in one place.

**Comment by WangQiangQuan** [comment]
ok，This is the error message “ abort
![1](https://github.com/getsentry/sentry-dart/assets/5965853/8a209316-429a-400a-978e-62c3d6b40b2f)
![2](https://github.com/getsentry/sentry-dart/assets/5965853/97ece9e5-5db5-4e66-9fe1-c662a78bae9b)
![3](https://github.com/getsentry/sentry-dart/assets/5965853/ae29a552-60aa-4eae-977c-9ef9d5b9a887)
”  on the sentry

**Comment by marandaneto** [comment]
@WangQiangQuan sorry the information you've given isn't enough to debug this, your sentry access is self-hosted so we cannot check the metadata, can you provide a minimal reproducible example? otherwise, we can't reproduce it.

**Comment by WangQiangQuan** [comment]
I don't know how to reproduce this, can you tell me how to get the metadata, and I'll get it from the self-hosted and give it to you

**Comment by marandaneto** [comment]
@WangQiangQuan https://github.com/getsentry/sentry-dart/issues/1539#issuecomment-1625119462
Please try this feature on sentry.io first, so I have access to the metadata.

**Comment by WangQiangQuan** [comment]
I haven't scheduled to put it on sentry.io recently, is there any other way to solve this problem？

**Comment by WangQiangQuan** [comment]
I haven't scheduled to put it on sentry.io recently, is there any other way to solve this problem？

**Comment by marandaneto** [comment]
> I haven't scheduled to put it on sentry.io recently, is there any other way to solve this problem？

No, I really need to dig into the metadata to check if this is something on our end.
Please provide a sentry.io issue link or a minimal reproducible example.
